File: src/main/java/cn/mycommons/tools/plugin/databindingconvert/util/IOUtil.java
package cn.mycommons.tools.plugin.databindingconvert.util;

import java.io.PrintWriter;

/**
 * IOUtil <br/>
 * Created by xiaqiulei on 2016-09-09.
 */
public class IOUtil {

    public static void writeToFile(String path, String content) {
        try {
            PrintWriter writer = new PrintWriter(path);
            writer.print(content);
            writer.flush();
            writer.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: src/main/java/cn/mycommons/tools/plugin/databindingconvert/util/IOUtil.kt
package cn.mycommons.tools.plugin.databindingconvert.util

import java.io.PrintWriter

/**
 * IOUtil <br></br>
 * Created by xiaqiulei on 2016-09-09.
 */
object IOUtil {

    fun writeToFile(path: String, content: String?) {
        try {
            val writer = PrintWriter(path)
            writer.print(content)
            writer.flush()
            writer.close()
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}